include/zephyr/kernel/thread.h

```diff
@@ -12,6 +12,7 @@
 #endif
 
 #include <zephyr/kernel/stats.h>
+#include <zephyr/sys/arch_interface.h>
```

Comment:
Was this meant to be included in the commit? Thought I would ask since it's not mentioned in the commit message